### PR TITLE
Jewel's Update

### PR DESCRIPTION
**Pull request by ShalomiAhavah**
Just added my profile info and also figured out how to do this

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -102,6 +102,17 @@ <h3>Lere</h3>
           </p>
         </article>
 
+        <article class="member">
+          <h3>Jewels</h3>
+          <p>Languages: HTML, CSS</p>
+          <p>GitHub: <a href="https://github.com/ShalomiAhavah">https://github.com/ShalomiAhavah</a></p>
+          </p>
```
Comment (josh76543210) [inline index.html]: Please remove the paragraph tags that you are not using. Thanks.

**Review by josh76543210 — CHANGES_REQUESTED** [review]
Hi @ShalomiAhavah, please remove the excess paragraph tag in your section. Looks good otherwise!

File: index.html (hunk 1 above)
Comment (ShalomiAhavah) [reply index.html]: Done(:

**Review by josh76543210 — APPROVED** [review]
Thanks @ShalomiAhavah

**Pull request merged**